src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -196,6 +196,74 @@ describe("Editor Page Component", () => {
         expect(saveProjectSpy).toBeCalledWith(expect.objectContaining(partialProject));
     });
 
+    describe("Editor Page Component Forcing Tag Scenario", () => {
```

Comment:
You likely don't need all of this setup code - It is duplicate from the other tests in the file.

Reply:
I've pushed a commit to simplify. Let me know if you have any questions.

Reply:
Yes, thank you.  See comments above